src/lib/services/move/module.ts

```diff
@@ -213,6 +213,8 @@ const zModuleInfoResponse = z
       .pick({ id: true, title: true })
       .nullable(),
     is_republished: z.boolean(),
+    recent_publish_block_height: z.number().nullable(),
```

Comment:
is this nullable?

Reply:
Will changed to `z.number().nonnegative()`